src/index.js

```diff
@@ -0,0 +1,131 @@
+//Wave 1
+"use strict";
+
+//Wave 2
+let currentTemp = 0;
+const tempNumber = document.getElementById('temp-number');
+const landscapeImg = document.getElementById('landscape-img');
+
+document.getElementById('increase-temp').addEventListener('click',()=>{
+    currentTemp += 1;
+    tempNumber.textContent = currentTemp;
+    changeTempColor();
+})
+
+document.getElementById('decrease-temp').addEventListener('click',()=>{
+    currentTemp -= 1;
+    tempNumber.textContent = currentTemp;
+    changeTempColor();
+})
```

Comment:
This works and is technically sound! That said, here are a couple of suggestions:

As mentioned earlier, your JS may run before the DOM is fully loaded, which could cause issues when selecting elements. To prevent this, consider wrapping your event listener registrations inside a `DOMContentLoaded` event handler. This ensures the DOM elements exist before you try to reference them.

For your event handlers, it's a good practice to define them as named functions and pass the function reference to `addEventListener`. This improves modularity by separating event registration from event logic, making your code easier to maintain and test.

**Note: This suggestion applies to all of your event registration logic.**

Below's suggestion is just for spacing.
```suggestion
document.getElementById('increase-temp').addEventListener('click',() => {
    currentTemp += 1;
    tempNumber.textContent = currentTemp;
    changeTempColor();
})

document.getElementById('decrease-temp').addEventListener('click',() => {
    currentTemp -= 1;
    tempNumber.textContent = currentTemp;
    changeTempColor();
})
```